docs/api/usage.rst

```diff
@@ -86,15 +86,15 @@ may be used:
         narrowed = Concrete.cast_as_concrete(model)
         reveal_type(narrowed) # Concrete1 | Concrete2 | Concrete3
 
-    def takes_model_cls(model: type[AbstractModel]) -> None:
-        narrowed = Concrete.cast_as_concrete(model)
+    def takes_model_cls(model_cls: type[AbstractModel]) -> None:
+        narrowed = Concrete.cast_as_concrete(model_cls)
         reveal_type(narrowed) # type[Concrete1] | type[Concrete2] | type[Concrete3]
 
-Note that at runtime this will raise an exception if the passed in object is either not a django
-model class/instance or is an abstract one.
+Note that at runtime this will raise an exception if the passed in object is
+either not a Django model class/instance or is an abstract one.
```

Comment:
Also capitalised `Django` to conform with the other ref, which is capitalised.

Reply:
capitalisation is a genuine weak point for me, my brain super doesn't notice it when I'm typing.